### PR TITLE
dextool: improve installation doc

### PR DESCRIPTION
**Pull request by christoffer-nylen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,37 +19,32 @@ The plugins in a standard installation of Dextool are:
...
-```sh
-# link https://dlang.org/download.html
-curl -fsS https://dlang.org/install.sh | bash -s dmd
-```
+Installation instructions for Ubuntu is available in the [doc/install](doc/install) directory.
+
+### Build and Install
 
-Once you have a D compiler, you also have access to the D package manager `dub`. The easiest way to run dextool is to do it via `dub`.
+The easiest way to run dextool is to do it via `dub`.
```
Comment (christoffer-nylen) [inline README.md:41]:
```suggestion
The easiest way to build and run dextool is to do it via `dub`.
```

File: README.md [hunk 2]
```diff
@@ -19,37 +19,32 @@ The plugins in a standard installation of Dextool are:
...
-Dextool depends on the following software packages:
+### Dependencies
 
- * [llvm](http://releases.llvm.org/download.html) (4.0+, both libclang and LLVM, see below)
-    * llvm-xyz-dev (4.0+)
-    * libclang-xyz-dev (4.0+)
- * [cmake](https://cmake.org/download) (3.5+)
- * [D compiler](https://dlang.org/download.html) (dmd 2.088.1+, ldc 1.16.0+)
- * [sqlite3](https://sqlite.org/download.html) (3.19.3-3+)
+To build and run dextool, you will need the following dependencies:
```
Comment (christoffer-nylen) [inline README.md:26]:
```suggestion
To build and run dextool, you will need the following packages:
```

File: README.md [hunk 3]
```diff
@@ -19,37 +19,32 @@ The plugins in a standard installation of Dextool are:
...
 
-Dextool has been tested with libclang [4.0, 5.0, 6.0, 7.0, 8.0].
+ * llvm >= 4.0
+ * llvm-xyz-dev >= 4.0
+ * libclang-xyz-dev >= 4.0
+ * cmake >= 3.5
+ * libsqlite3-dev >= 3.19.3-3
```
Comment (christoffer-nylen) [inline README.md:32]:
```suggestion
 * [sqlite3](https://sqlite.org/download.html) >= 3.19.3-3
```

**Comment by joakim-brannstrom** [comment]
I would like the ubuntu file would be named `UBUNTU.md` because the instructions are applicable to all ubuntu versions.

File: README.md [hunk 4]
```diff
@@ -19,37 +19,32 @@ The plugins in a standard installation of Dextool are:
...
- * [sqlite3](https://sqlite.org/download.html) (3.19.3-3+)
+To build and run dextool, you will need the following packages:
 
-Dextool has been tested with libclang [4.0, 5.0, 6.0, 7.0, 8.0].
+ * [llvm](http://releases.llvm.org/download.html) >= 4.0
+ * llvm-xyz-dev >= 4.0
+ * libclang-xyz-dev >= 4.0
+ * [cmake](https://cmake.org/download) >= 3.5
+ * [sqlite3](https://sqlite.org/download.html) >= 3.19.3-3
+ * [D compiler](https://dlang.org/download.html) (dmd >= 2.088.1 or ldc >= 1.16.0+)
```
Comment (joakim-brannstrom) [inline README.md:33]: Remove the `+` after 1.16.0

**Comment by joakim-brannstrom** [comment]
thank you

**Pull request merged**